modules/controller.js

```diff
@@ -0,0 +1,78 @@
+import { listOfBooks } from './selectors.js';
+
+const displayBooks = (booksArray) => {
+  const collection = JSON.parse(localStorage.getItem('books'));
+  booksArray.books = collection;
+
+  booksArray.books.forEach((book) => {
```

Comment:
- [ ] I'm unable to add or remove books due to app throwing an error in runtime from the browser console, as can be seen below. `bookArray` appears to be null when the program starts as it does not hold any book initially. I suggest that you try wrapping your enumerable around a conditional block.

**Load Error:**
![image](https://user-images.githubusercontent.com/12745474/214255096-225ee79b-e2b1-467a-87c0-c986b9c67fdf.png)